extensions/guacamole-auth-ldap/src/main/java/org/apache/guacamole/auth/ldap/user/UserService.java

```diff
@@ -84,14 +84,17 @@
      */
     private void putAllUsers(Map<String, User> users, LDAPConnection ldapConnection,
             String usernameAttribute) throws GuacamoleException {
-
+    
```

Comment:
More generally, we should avoid purely-whitespace changes. There's no reason for this line to be modified, let alone just for the sake of adding an unused indent.